.env.example

```diff
@@ -2,9 +2,9 @@ DB_NAME=database_name
 DB_USER=database_user
 DB_PASSWORD=database_password
 
-# Or you can use a DSN (database source name) 
-# In this case you can leave empty variables DB_NAME, DB_USER, DB_PASSWORD, DB_HOST 
-# DATABASE_URL=mysql://database_user:database_password@database_host:database_port/database_name
+# Optionally, you can use a data source name (DSN)
+# When using a DSN, you can leave DB_NAME, DB_USER, DB_PASSWORD, and DB_HOST empty
```

Comment:
Actually, with my latest changes, you can change to: 
```suggestion
# When using a DSN, you can remove DB_NAME, DB_USER, DB_PASSWORD, and DB_HOST variables
```

Reply:
With the current implementation, the existence of the DB_ variables isn't checked so removing them would throw an error. We would need to add a null coalesce operator.